tests/conftest.py

```diff
@@ -5,16 +5,18 @@
 from web3 import Web3
```

Comment:
For better typing
```suggestion
from src.web3_extentions.typings import Web3
```

Reply:
And also, `Web3` doesn't have `lido_validators` attribute. Could you add it?

Reply:
Fixed